### PR TITLE
added warning

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0623528cbd/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-dd2c0165b1/how-to/).

File: content/rackspace-email/secure-existing-webmail-private-label-site.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 created_by: Ben Smith
-last_modified_date: '2018-02-23'
+last_modified_date: '2018-03-07'
 last_modified_by: Cat Lookabaugh
 product: Rackspace Email
 product_url: rackspace-email
 ---
 
-Webmail private label sites created before September 2017 were deployed to an environment that did not enforce the use of secure web protocols over HTTPS, such as SSL or TLS. To add this additional level of protection to your existing Webmail private label site, we recommend you secure it.
+Webmail private label sites created before September, 2017 were deployed to an environment that did not enforce the use of secure web protocols over HTTPS, such as SSL or TLS. To add this additional level of protection to your existing Webmail private label site, we recommend you secure it.
```
Comment (stephamon) [inline content/rackspace-email/secure-existing-webmail-private-label-site.md]: I think you only need a comma after September if you're including a specific date.

Comment (catlook) [reply content/rackspace-email/secure-existing-webmail-private-label-site.md]: ok...looked weird without it...but i'll pull!

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-abb3bd2ded/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**